charts/kubescape-operator/templates/kubescape-scheduler/cronjob.yaml

```diff
@@ -2,7 +2,7 @@
   {{- fail "`batch/v1/CronJob not supported`" }}
 {{- end }}
 {{- $config := fromYaml (include "config" .) }}
-{{- if and .Values.kubescapeScheduler.enabled $config.kubescape.enabled }}
+{{- if and $config.gateway.enabled $config.kubescape.enabled }}
```

Comment:
gateway?

Reply:
yes, because we said scheduled scans only make sense when we have Armo platform